json/src/ser.rs

```diff
@@ -489,44 +489,44 @@ impl<'a, W, F> ser::Serializer for MapKeySerializer<'a, W, F>
         Err(Error::Syntax(ErrorCode::KeyMustBeAString, 0, 0))
     }
 
-    fn serialize_isize(&mut self, _value: isize) -> Result<()> {
-        Err(Error::Syntax(ErrorCode::KeyMustBeAString, 0, 0))
+    fn serialize_isize(&mut self, value: isize) -> Result<()> {
+        self.ser.serialize_str(&value.to_string())
```

Comment:
That allocates on the heap, can you find a way around that? Like write a `"`, the serialize normally, write another quote?